src/plugins/ui_actions/public/service/ui_actions_service.ts

```diff
@@ -154,10 +149,7 @@ export class UiActionsService {
     return actions as Action[];
   };
 
-  public readonly getTriggerCompatibleActions = async (
-    triggerId: string,
-    context: object
-  ): Promise<Action[]> => {
+  public readonly getTriggerCompatibleActions = async (triggerId: string, context: object) => {
```

Comment:
Same here, let's make it `Action` explicit

Reply:
Done.